doc/ref/start-of-release.md

```diff
@@ -0,0 +1,43 @@
+% Explanation of start of release changes
+
+## Overview
```

Comment:
Is there just one section, titled "Overview", or will there be more? If the former, consider omitting this heading.